templates/keycloak/files/.platform/services.yaml

```diff
@@ -0,0 +1,12 @@
+database:
+    type: mariadb:10.4
+    disk: 1024
+    configuration:
+        schemas:
+            - main
+        endpoints:
+            mariadb:
+                default_schema: main
+                privileges:
+                    main: admin
```

Comment:
This is the default behavior already, so why is it being spelled out?  If it's just to force a password to be generated, let's put that in a comment.

Reply:
It is to force the password.

Reply:
Done.